azurerm/resource_arm_sql_virtual_network_rule_test.go

```diff
@@ -0,0 +1,473 @@
+	(This test configuration is intended to succeed.)
+*/
+func testAccAzureRMSqlVirtualNetworkRule_multipleSubnets(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+    name = "acctestRG_%d"
+    location = "%s"
+}
+resource "azurerm_virtual_network" "vnet1" {
+  name                = "acctestvnet1%d"
+  address_space       = ["10.7.29.0/24"]
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+resource "azurerm_virtual_network" "vnet2" {
+  name                = "acctestvnet2%d"
+  address_space       = ["10.1.29.0/29"]
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+resource "azurerm_subnet" "vnet1_subnet1" {
+  name = "acctestsubnet1%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  virtual_network_name = "${azurerm_virtual_network.vnet1.name}"
+  address_prefix = "10.7.29.0/29"
+  service_endpoints = ["Microsoft.Sql"]
+}
+resource "azurerm_subnet" "vnet1_subnet2" {
+  name = "acctestsubnet2%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  virtual_network_name = "${azurerm_virtual_network.vnet1.name}"
+  address_prefix = "10.7.29.128/29"
+  service_endpoints = ["Microsoft.Sql"]
+}
+resource "azurerm_subnet" "vnet2_subnet1" {
+  name = "acctestsubnet3%d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  virtual_network_name = "${azurerm_virtual_network.vnet2.name}"
+  address_prefix = "10.1.29.0/29"
+  service_endpoints = ["Microsoft.Sql"]
+}
+resource "azurerm_sql_server" "test" {
+    name = "acctestsqlserver1%d"
+    resource_group_name = "${azurerm_resource_group.test.name}"
+    location = "${azurerm_resource_group.test.location}"
+    version = "12.0"
+    administrator_login = "missadmin"
+    administrator_login_password = "${md5(%d)}!"
+}
+resource "azurerm_sql_virtual_network_rule" "rule1" {
+    name = "acctestsqlvnetrule1%d"
+    resource_group_name = "${azurerm_resource_group.test.name}"
+    server_name = "${azurerm_sql_server.test.name}"
+    virtual_network_subnet_id = "${azurerm_subnet.vnet1_subnet1.id}"
+    ignore_missing_vnet_service_endpoint = false
+}
+resource "azurerm_sql_virtual_network_rule" "rule2" {
+    name = "acctestsqlvnetrule2%d"
+    resource_group_name = "${azurerm_resource_group.test.name}"
+    server_name = "${azurerm_sql_server.test.name}"
+    virtual_network_subnet_id = "${azurerm_subnet.vnet1_subnet2.id}"
+    ignore_missing_vnet_service_endpoint = false
+}
+resource "azurerm_sql_virtual_network_rule" "rule3" {
+    name = "acctestsqlvnetrule3%d"
+    resource_group_name = "${azurerm_resource_group.test.name}"
+    server_name = "${azurerm_sql_server.test.name}"
+    virtual_network_subnet_id = "${azurerm_subnet.vnet2_subnet1.id}"
+    ignore_missing_vnet_service_endpoint = false
+}
```

Comment:
_minor_ could we make the spacing consistent here with 2 spaces for indents?

Reply:
👍